plutus-core/untyped-plutus-core/src/UntypedPlutusCore.hs

```diff
@@ -1,13 +1,13 @@
 module UntypedPlutusCore (
-    module Export
-    , Term (..)
-    , Program (..)
-    , applyProgram
-    , parseScoped
-    , PLC.DefaultUni
-    , PLC.DefaultFun
-    , mkDefaultProg
-    ) where
+    module Export,
+    Term (..),
+    Program (..),
+    applyProgram,
+    parseScoped,
+    PLC.DefaultUni,
+    PLC.DefaultFun,
+    mkDefaultProg,
+) where
```

Comment:
Why is ths format change? I like more the old one :)

Reply:
I think I ran fourmolu on it..